src/paperless/settings.py

```diff
@@ -47,7 +47,11 @@
 if _allowed_hosts:
     ALLOWED_HOSTS = _allowed_hosts.split(",")
 
-
+FORCE_SCRIPT_NAME = None
+_force_script_name = os.getenv("PAPERLESS_FORCE_SCRIPT_NAME")
+if _force_script_name:
+    FORCE_SCRIPT_NAME = _force_script_name
```

Comment:
Looking at https://docs.python.org/3/library/os.html#os.getenv, you could just write `FORCE_SCRIPT_NAME = os.getenv("PAPERLESS_FORCE_SCRIPT_NAME")` as this would return `None` per default. There's no need to set it to `None` explicitly.